### PR TITLE
Add custom bool normalizer and update models

### PR DESCRIPTION
**Pull request by Gaitholabi**

### CODE REVIEW

File: src/Model/Core/Brand.php [hunk 1]
```diff
@@ -215,7 +215,7 @@ class Brand extends BaseModel
     private $languageToggle;
 
     /**
-     * @var int
+     * @var bool
      * @SerializedName("operator_template_mode")
      */
     private $operatorTemplateMode;
```
Comment (bytestream) [inline src/Model/Core/Brand.php:221]: Same as `frontend_template_mode`

File: src/Model/User/User.php [hunk 2]
```diff
@@ -110,7 +110,7 @@ class User extends BaseModel
     private $notes;
 
     /**
-     * @var int|null
+     * @var bool|null
      * @SerializedName("organisation_access_level")
```
Comment (bytestream) [inline src/Model/User/User.php:114]: While current values are `0` or `1` I'm not sure that this should be boolean. @jshah4517 ?

Comment (jshah4517) [reply src/Model/User/User.php:114]: This makes more sense as just an int, it's 0` - Manager, `1` - User.

File: src/Model/User/User.php [hunk 3]
```diff
@@ -499,18 +499,18 @@ public function setFacebookId(?int $facebookId): self
     }
 
     /**
-     * @return int|null
+     * @return bool|null
      */
-    public function getOrganisationAccessLevel(): ?int
+    public function getOrganisationAccessLevel(): ?bool
```
Comment (jshah4517) [inline src/Model/User/User.php:504]: Same applies here as above comment.

File: src/Model/User/User.php [hunk 4]
```diff
@@ -499,18 +499,18 @@ public function setFacebookId(?int $facebookId): self
...
         return $this->organisationAccessLevel;
     }
 
     /**
-     * @param int|null $organisationAccessLevel
+     * @param bool|null $organisationAccessLevel
      * @return self
      */
-    public function setOrganisationAccessLevel(?int $organisationAccessLevel): self
+    public function setOrganisationAccessLevel(?bool $organisationAccessLevel): self
```
Comment (jshah4517) [inline src/Model/User/User.php:513]: Same applies here as above comment.

File: test/DataFixtures/Ticket/EmailData.php [hunk 5]
```diff
@@ -23,7 +23,7 @@ class EmailData extends BaseModelData
         'delete_downloaded' => null,
         'created_at' => 0,
         'updated_at' => 0,
-        'oauth' => 1,
+        'oauth' => '1',
```
Comment (jshah4517) [inline test/DataFixtures/Ticket/EmailData.php:26]: Better if this is a random string, it's essentially an encrypted string like a password.

Comment (Gaitholabi) [reply test/DataFixtures/Ticket/EmailData.php:26]: this doesn't really matter, there's no logical value beyond the type being checked

**Review by bytestream — APPROVED** [review]
LGTM 👍

**Review by jshah4517 — APPROVED** [review]

**Pull request merged**